recipes/qt/5.x.x/conanfile.py

```diff
@@ -1227,6 +1271,8 @@ def _create_plugin(pluginname, libname, type, requires):
                     self.cpp_info.components[component].exelinkflags.extend(obj_files)
                     self.cpp_info.components[component].sharedlinkflags.extend(obj_files)
 
+        self.cpp_info.set_property("cmake_build_modules", build_modules)
```

Comment:
why isn't this done per-component ?

Reply:
https://github.com/conan-io/conan/issues/10282